pluggy/callers.py

```diff
@@ -41,10 +41,21 @@ def from_call(cls, func):
         return cls(result, excinfo)
 
     def force_result(self, result):
+        """Force the result(s) to ``result``.
+
+        If the hook was marked as a ``firstresult`` a single value should
+        be set otherwise set a (modified) list of results. Any exceptions
+        found during invocation will be deleted.
+        """
         self.result = result
```

Comment:
Not related to the PR: should we make all attributes private?

Reply:
Hmm yeah maybe?
I don't think the original `_CallOutcome` did this any differently fwiw.

Reply:
Oh definitely, this wasn't a criticism of the current code, just something I thought I might bring up. 

If we want to have a well-defined API we might want to start doing that somewhat sooner; a lot of problems of the pytest API stem from the fact that public attributes should never been so, but changing them later is a pain because client code now depends on it (even if by accident).

Reply:
Yeah totally agree.
I just wasn't sure if `.result` was initially supposed to be public.
I'm cool to switch it in this PR as long as you guys are.